source/docs/articles/frequently-asked-questions.md

```diff
@@ -9,25 +9,27 @@ keywords: getting started, faqs, sites, pantheon, plans, developing, security
 
 ### Can I put production sites on Pantheon?
 
-Yes. Hundreds of live production sites run on Pantheon.
+Yes. Thousands of live production sites run on Pantheon.
 
 ### What versions of Drupal does Pantheon support?
 
 Pantheon supports Drupal 6 and 7, as well as development sandboxes for Drupal 8.
 
-## What versions of WordPress does Pantheon support?
+### What versions of WordPress does Pantheon support?
 
 Pantheon supports the most recent release of WordPress via [upstream](https://github.com/pantheon-systems/WordPress), which includes platform integration plugins and a pre-configured wp-config.php.
 
 ### How much does Pantheon cost?
 
-Pantheon is free for developers. Our live site plans currently start as low as $25 monthly for personal sites, and $100 for professional sites. Learn more on [our pricing page](https://getpantheon.com/pricing).
+Pantheon is free for developers. Our live site plans currently start as low as $25 monthly for personal sites, and $100 for professional sites. Learn more on our [pricing page](https://getpantheon.com/pricing).
 
 ### Where are the Pantheon servers located?
 
 All Pantheon servers are currently located in the United States. We have plans to expand to Europe, but we don't have an ETA for when they will be available for end-users.
 
-### Can I run non Drupal applications on Pantheon?
+You can use a [CDN](/docs/articles/drupal/content-delivery-network-cdn-for-file-distribution/) for rapidly serving files from multiple locations, and Pantheon is faster than the time savings from having a local server.
```

Comment:
What does this mean "...and Pantheon is faster than the time savings from having a local server."